app/models/recipe.rb

```diff
@@ -0,0 +1,7 @@
+class Recipe < ApplicationRecord
+  belongs_to :user
+  has_many :recipe_foods
+  has_many :foods, through: :recipe_foods
+
```

Comment:
Great job implementing the migrations and models, I will suggest we remove the third association as the recipe_foods are already linked to a particular food, so the association is automatically created with the Recipe.

Reply:
Noted